docs/src/stories/Options/index.js

```diff
@@ -66,6 +65,7 @@ class WithOptions extends PureComponent {
               <option value="multiSelect">Multi select</option>
               <option value="simpleSelect">Simple select</option>
               <option value="radioSelect">Radio select</option>
+              <option value="hierarchical">Hierarchical</option>
```

Comment:
You beat me to it! And here I was wondering what the heck is he talking about - stories is already updated! 😆